feg/gateway/services/session_proxy/credit_control/gx/gx_client.go

```diff
@@ -244,8 +245,18 @@ func (gxClient *GxClient) createCreditControlMessage(
 	}
 
 	apn := datatype.UTF8String(request.Apn)
-	if globalConfig != nil && len(globalConfig.PCFROverwriteApn) > 0 {
-		apn = datatype.UTF8String(globalConfig.PCFROverwriteApn)
+	if globalConfig != nil {
```

Comment:
same here, this is a function like `getDefaultFramedIpv4Addr`
Not dividing pieces of the code into functions cause the code to be illegible